### PR TITLE
Updated naming convention for functions

### PR DESCRIPTION
**Pull request by sshubhamagg**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,3 +23,7 @@
 ##### Updated Contract Address
 
 * Updated the contract addresses as per the new deployments on ropsten.
+#### 1.0.5 (2020-11-19)
+
+##### Updated usage of handlename to Inblox ID
```
Comment (apoorvq) [inline CHANGELOG.md:28]: Should be "Updated naming convention from Inblox Handle Name to InbloxID".

File: src/index.js [hunk 2]
```diff
@@ -88,64 +88,64 @@ class InbloxHandlename {
...
-    if (addressOfHandlename !== '0x0000000000000000000000000000000000000000') {
-      return HANDLENAME_ALREADY_TAKEN;
+    if (addressOfInbloxId !== '0x0000000000000000000000000000000000000000') {
+      return INBLOXID_ALREADY_TAKEN;
     }
 
-    const fees = await this.handlenameFees();
-    const isHNValid = await isHandlenameValid(handleName);
+    const fees = await this.inbloxIdFees();
+    const isHNValid = await isInbloxIdValid(inbloxId);
```
Comment (ARJUN-R34) [inline src/index.js:141]: `isHNValid` should also get updated according to the convention.

File: src/index.js [hunk 3]
```diff
@@ -158,36 +158,36 @@ class InbloxHandlename {
...
+    if (addressOfInbloxId !== '0x0000000000000000000000000000000000000000') {
+      return INBLOXID_ALREADY_TAKEN;
     }
 
-    const updateCount = await this.handlenameUpdateCount(userAddress);
-    const fees = await this.handlenameFees();
-    const isHNValid = await isHandlenameValid(newHandleName);
+    const updateCount = await this.getUpdateCount(userAddress);
+    const fees = await this.inbloxIdFees();
+    const isHNValid = await isInbloxIdValid(newInbloxId);
```
Comment (ARJUN-R34) [inline src/index.js:181]: Update `isHNValid` variable according to the latest convention.

**Review by apoorvq — APPROVED** [review]

**Comment by apoorvq** [comment]
Merging for Naming Conversation Changes.

**Pull request merged**